### ISSUE TITLE
Travis CI failed for a fresh master fork

### ISSUE THREAD

**Issue by Tailp** [Spirals-Team/repairnator#867]
Hi, I test making a PR with only a comment line changed after forking from master branch and Travis failed while it's supposed to be passing . Maybe it's an issue on my part ?. Please have a look :) . 

Related PR : (#866)

Output: (more details are in this [Travis raw output log](https://api.travis-ci.org/v3/job/558999668/log.txt)):
```
Tests run: 9, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 113.068 sec <<< FAILURE! - in fr.inria.spirals.repairnator.scanner.ProjectScannerTest
testGetBuildToBeInspectedWithPassingWithPreviousPassingFromPR(fr.inria.spirals.repairnator.scanner.ProjectScannerTest)  Time elapsed: 33.525 sec  <<< FAILURE!
java.lang.AssertionError: expected:<BuildToBeInspected{patchedBuild=Build{number='2774', state='PASSED', duration=445, eventType='PULL_REQUEST', previousState='PASSED', pullRequestTitle='review 2: Change local variable name refactoring', pullRequestNumber=1005, startedAt=Wed Mar 15 19:19:41 UTC 2017, finishedAt=Wed Mar 15 19:27:06 UTC 2017, repository=Repository{name='spoon', slug='INRIA/spoon', active=false, privateProperty=false, description='null', githubLanguage='null', starred=false, owner=null, defaultBranch=null}, branch=Branch{name='master', repository=null, defaultBranch=false, existsOnGithub=false, lastBuild=null}, commit=Commit{sha='74cc445c5cbd55cddb02164fa9ecee25c6f6d6d0', ref='refs/pull/1005/merge', message='feature: change LocalVariable name', compareUrl='https://github.com/INRIA/spoon/pull/1005', committedAt=Wed Mar 15 19:17:45 UTC 2017, committer=null, author=null}, jobs=Job{allowFailure=false, number='2774.1', state=PASSED, startedAt=Wed Mar 15 19:19:41 UTC 2017, finishedAt=Wed Mar 15 19:27:06 UTC 2017, build=Build{number='2774', state='PASSED', duration=445, eventType='PULL_REQUEST', previousState='PASSED', pullRequestTitle='review 2: Change local variable name refactoring', pullRequestNumber=1005, startedAt=Wed Mar 15 19:19:41 UTC 2017, finishedAt=Wed Mar 15 19:27:06 UTC 2017, repository=null, branch=null, commit=null, jobs=, updatedAt=null, tag='null', createdBy=null}, queue='builds.gce', repository=Repository{name='spoon', slug='INRIA/spoon', active=false, privateProperty=false, description='null', githubLanguage='null', starred=false, owner=null, defaultBranch=null}, commit=Commit{sha='74cc445c5cbd55cddb02164fa9ecee25c6f6d6d0', ref='refs/pull/1005/merge', message='feature: change LocalVariable name', compareUrl='https://github.com/INRIA/spoon/pull/1005', committedAt=Wed Mar 15 19:17:45 UTC 2017, committer=null, author=null}, owner=Owner{type=ORGANIZATION, login='INRIA', name='null', githubId=0, avatarUrl='null', isSyncing=false, syncedAt=null}, createdAt=Wed Mar 15 19:19:03 UTC 2017, updatedAt=Sat Jun 02 03:00:07 UTC 2018, log=fr.inria.jtravis.entities.Log@eddf281b, buildTool=null}, updatedAt=Thu Apr 11 10:57:12 UTC 2019, tag='null', createdBy=Owner{type=USER, login='pvojtechovsky', name='null', githubId=0, avatarUrl='null', isSyncing=false, syncedAt=null}}, buggyBuild=Build{number='2741', state='PASSED', duration=464, eventType='PULL_REQUEST', previousState='PASSED', pullRequestTitle='WiP: Change local variable name refactoring', pullRequestNumber=1005, startedAt=Sat Mar 11 17:53:01 UTC 2017, finishedAt=Sat Mar 11 18:00:45 UTC 2017, repository=Repository{name='spoon', slug='INRIA/spoon', active=false, privateProperty=false, description='null', githubLanguage='null', starred=false, owner=null, defaultBranch=null}, branch=Branch{name='master', repository=null, defaultBranch=false, existsOnGithub=false, lastBuild=null}, commit=Commit{sha='203c0973905fd932d763cd50049c1dadc73da4bc', ref='refs/pull/1005/merge', message='feature: change LocalVariable name', compareUrl='https://github.com/INRIA/spoon/pull/1005', committedAt=Sat Mar 11 17:50:36 UTC 2017, committer=null, author=null}, jobs=Job{allowFailure=false, number='null', state=null, startedAt=null, finishedAt=null, build=null, queue='null', repository=null, commit=null, owner=null, createdAt=null, updatedAt=null, log=null, buildTool=null}, updatedAt=Thu Apr 11 10:57:12 UTC 2019, tag='null', createdBy=Owner{type=USER, login='pvojtechovsky', name='null', githubId=0, avatarUrl='null', isSyncing=false, syncedAt=null}}, status=PASSING_AND_PASSING_WITH_TEST_CHANGES, runId='test'}> but was:<null>
	at fr.inria.spirals.repairnator.scanner.ProjectScannerTest.testGetBuildToBeInspectedWithPassingWithPreviousPassingFromPR(ProjectScannerTest.java:158)

Results :
Failed tests: 
  ProjectScannerTest.testGetBuildToBeInspectedWithPassingWithPreviousPassingFromPR:158 expected:<BuildToBeInspected{patchedBuild=Build{number='2774', state='PASSED', duration=445, eventType='PULL_REQUEST', previousState='PASSED', pullRequestTitle='review 2: Change local variable name refactoring', pullRequestNumber=1005, startedAt=Wed Mar 15 19:19:41 UTC 2017, finishedAt=Wed Mar 15 19:27:06 UTC 2017, repository=Repository{name='spoon', slug='INRIA/spoon', active=false, privateProperty=false, description='null', githubLanguage='null', starred=false, owner=null, defaultBranch=null}, branch=Branch{name='master', repository=null, defaultBranch=false, existsOnGithub=false, lastBuild=null}, commit=Commit{sha='74cc445c5cbd55cddb02164fa9ecee25c6f6d6d0', ref='refs/pull/1005/merge', message='feature: change LocalVariable name', compareUrl='https://github.com/INRIA/spoon/pull/1005', committedAt=Wed Mar 15 19:17:45 UTC 2017, committer=null, author=null}, jobs=Job{allowFailure=false, number='2774.1', state=PASSED, startedAt=Wed Mar 15 19:19:41 UTC 2017, finishedAt=Wed Mar 15 19:27:06 UTC 2017, build=Build{number='2774', state='PASSED', duration=445, eventType='PULL_REQUEST', previousState='PASSED', pullRequestTitle='review 2: Change local variable name refactoring', pullRequestNumber=1005, startedAt=Wed Mar 15 19:19:41 UTC 2017, finishedAt=Wed Mar 15 19:27:06 UTC 2017, repository=null, branch=null, commit=null, jobs=, updatedAt=null, tag='null', createdBy=null}, queue='builds.gce', repository=Repository{name='spoon', slug='INRIA/spoon', active=false, privateProperty=false, description='null', githubLanguage='null', starred=false, owner=null, defaultBranch=null}, commit=Commit{sha='74cc445c5cbd55cddb02164fa9ecee25c6f6d6d0', ref='refs/pull/1005/merge', message='feature: change LocalVariable name', compareUrl='https://github.com/INRIA/spoon/pull/1005', committedAt=Wed Mar 15 19:17:45 UTC 2017, committer=null, author=null}, owner=Owner{type=ORGANIZATION, login='INRIA', name='null', githubId=0, avatarUrl='null', isSyncing=false, syncedAt=null}, createdAt=Wed Mar 15 19:19:03 UTC 2017, updatedAt=Sat Jun 02 03:00:07 UTC 2018, log=fr.inria.jtravis.entities.Log@eddf281b, buildTool=null}, updatedAt=Thu Apr 11 10:57:12 UTC 2019, tag='null', createdBy=Owner{type=USER, login='pvojtechovsky', name='null', githubId=0, avatarUrl='null', isSyncing=false, syncedAt=null}}, buggyBuild=Build{number='2741', state='PASSED', duration=464, eventType='PULL_REQUEST', previousState='PASSED', pullRequestTitle='WiP: Change local variable name refactoring', pullRequestNumber=1005, startedAt=Sat Mar 11 17:53:01 UTC 2017, finishedAt=Sat Mar 11 18:00:45 UTC 2017, repository=Repository{name='spoon', slug='INRIA/spoon', active=false, privateProperty=false, description='null', githubLanguage='null', starred=false, owner=null, defaultBranch=null}, branch=Branch{name='master', repository=null, defaultBranch=false, existsOnGithub=false, lastBuild=null}, commit=Commit{sha='203c0973905fd932d763cd50049c1dadc73da4bc', ref='refs/pull/1005/merge', message='feature: change LocalVariable name', compareUrl='https://github.com/INRIA/spoon/pull/1005', committedAt=Sat Mar 11 17:50:36 UTC 2017, committer=null, author=null}, jobs=Job{allowFailure=false, number='null', state=null, startedAt=null, finishedAt=null, build=null, queue='null', repository=null, commit=null, owner=null, createdAt=null, updatedAt=null, log=null, buildTool=null}, updatedAt=Thu Apr 11 10:57:12 UTC 2019, tag='null', createdBy=Owner{type=USER, login='pvojtechovsky', name='null', githubId=0, avatarUrl='null', isSyncing=false, syncedAt=null}}, status=PASSING_AND_PASSING_WITH_TEST_CHANGES, runId='test'}> but was:<null>

Tests run: 9, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] repairnator-root ................................... SUCCESS [ 10.641 s]
[INFO] repairnator-core ................................... SUCCESS [ 14.579 s]
[INFO] repairnator-pipeline ............................... SUCCESS [26:40 min]
[INFO] repairnator-scanner ................................ FAILURE [01:54 min]
[INFO] repairnator-dockerpool ............................. SKIPPED
[INFO] repairnator-checkbranches .......................... SKIPPED
[INFO] repairnator-realtime ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 29:06 min
[INFO] Finished at: 2019-07-15T16:39:55Z
[INFO] Final Memory: 184M/698M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project repairnator-scanner: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/travis/build/Spirals-Team/repairnator/repairnator/repairnator-scanner/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :repairnator-scanner
travis_time:end:0b59b88f:start=1563207046635614720,finish=1563208795894318050,duration=1749258703330
[0K[31;1mThe command "./.travis/travis-run.sh" exited with 1.[0m

Done. Your build exited with 1.
```

**Comment by Tailp** [comment]
Also related (travis build)[https://travis-ci.org/Spirals-Team/repairnator/jobs/559018886] where I just removed the comment right after the previous build.

**Comment by btellstrom** [comment]
Closed after discussion in #866

**Comment by monperrus** [comment]
Duplicate of #744, which is becoming high priority
